pkg/managed/secrets.go

```diff
@@ -177,6 +177,15 @@ func newSecrets(mbPair *types.ModelBindingPair, managedCluster *types.ManagedClu
 		}
 
 		secrets = append(secrets, secretObj)
+
+		// VZ-1596: Create secrets for configured list of arbitrary secrets
+		for _, secretName = range domain.Spec.Configuration.Secrets {
+			if secretObj, err = newSecret(secretName, namespace, kubeClientSet, nil, nil); err == nil {
+				secrets = append(secrets, secretObj)
+			} else {
+				zap.S().Errorf("Copying secret %s to namespace %s is giving error %v", secretName, namespace, err)
```

Comment:
This is a nit.  Are you copying the secret or just generating a new secret?

Reply:
Also, the comment in the outer if statement is "// Each WebLogic domain requires a runtime encryption secret that contains a randomly generated password."  This comment should probably be moved inside the outer if statement since more is being done now in the if statement

Reply:
I copied that error text from a section above where it's doing something very similar, so if the text is inaccurate then it needs to be fixed in the existing code too.

Regarding moving the outer comment, agreed, I'll make that change.

Reply:
Sounds good